src/components/Node.js

```diff
@@ -7,15 +7,31 @@ export default class Node extends React.Component {
   constructor(props) {
     super(props);
     this.colorScale = scale.category10();
-    this.state = {loading: true, chartWidth: 300, chartHeigth: this.props.chartHeigth || 50};
+    this.state = {
+      loading: true,
+      chartWidth: 400,
+      chartHeigth: 20,
+      expanded : false,
+      small: false,
+    };
   }
 
   componentDidMount() {
-    // Update chart width
+    //Update chart width
     setInterval(() => {
-      let value = this.panel.offsetWidth-50;
-      if (this.state.chartWidth != value) {
-        this.setState({chartWidth: value});
+      let value = this.panel.offsetWidth;
+      if (window.innerWidth < 1150 && !this.state.small) {
+        this.setState({small: true});
+      } else if (window.innerWidth >= 1150 && this.state.small) {
+        this.setState({small: false, expanded : false});
+      }
+      if (this.state.chartWidth != value/3 && !this.state.small ||
+          this.state.chartWidth != value/20 && this.state.small ) {
+        if (this.state.small) {
+          this.setState({chartWidth: value/20});
+        } else {
+          this.setState({chartWidth: value/3});
```

Comment:
Currently bars in the chars are too thick. They should be the same as in other charts like ledger close time and tx&ops charts. To achieve this let's make the chart wider (`value/2`) and change backend API to return 150 measurements instead of 70 (one-line change in `app.js`).